pkg/printers/humanreadable.go

```diff
@@ -279,6 +281,11 @@ func printHeader(columnNames []string, w io.Writer) error {
 
 // PrintObj prints the obj in a human-friendly format according to the type of the obj.
 func (h *HumanReadablePrinter) PrintObj(obj runtime.Object, output io.Writer) error {
+	return h.PrintObjEvent("", obj, output)
+}
+
+// PrintObj prints the obj in a human-friendly format according to the type of the obj.
```

Comment:
Why not just pass Event to PrintObj and have it to do the right thing?  This feels like a workaround to the underlying structure, and events are runtime.Objects just like everything else.